### PR TITLE
Add script for remove ledger

### PR DESCRIPTION
**Pull request by adenishchenko**
Signed-off-by: anton.denishchenko <anton.denishchenko@evernym.com>

### CODE REVIEW

File: scripts/remove_ledger.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import shutil
+from sys import argv
+script, file_path = argv
+from indy_common.config_util import getConfig
+from indy_common.config_helper import ConfigHelper
+
+config = getConfig()
+config_helper = ConfigHelper(config)
+
+def main(file_path):
```
Comment (askolesov) [inline scripts/remove_ledger.py:13]: Let's rename `file_parth` to `ledger_name`. What do you think? It's not a path actually.

File: scripts/remove_ledger.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    exceptions = ["domen", "config", "pool", "audit"]
+    if file_path not in exceptions:
+        for path in Path(config_helper.ledger_data_dir).rglob(file_path + "_*"):
+            shutil.rmtree(str(path))
+            print('follow directory was deleted: ' + path.name)
+    else:
+        print('Can`t delete ledger: ' + file_path)
```
Comment (askolesov) [inline scripts/remove_ledger.py:20]:
```suggestion
        print('Can`t delete built in ledger: ' + file_path)
```

File: scripts/remove_ledger.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+config = getConfig()
+config_helper = ConfigHelper(config)
+
+def main(file_path):
+    exceptions = ["domen", "config", "pool", "audit"]
+    if file_path not in exceptions:
+        for path in Path(config_helper.ledger_data_dir).rglob(file_path + "_*"):
+            shutil.rmtree(str(path))
+            print('follow directory was deleted: ' + path.name)
```
Comment (askolesov) [inline scripts/remove_ledger.py:18]:
```suggestion
            print('The follow directory was deleted: ' + path.name)
```

**Review by askolesov — CHANGES_REQUESTED** [review]
Looks good!

**Comment by askolesov** [comment]
(ci) test this please

File: scripts/remove_ledger.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+from pathlib import Path
+import shutil
+from sys import argv
+from indy_common.config_util import getConfig
+from indy_common.config_helper import ConfigHelper
+
+
+def remove(ledger_name):
+    exceptions = ["domen", "config", "pool", "audit"]
```
Comment (WadeBarnes) [inline scripts/remove_ledger.py:11]: Should `domen` be `domain`?

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
I'd highly recommend the script provide some feedback to the user and wait for confirmation. 
 It should clearly indicate it's about to delete a ledger, which ledger it's about to delete, describe the consequences, and then wait for confirmation from the user.

Some help text for the script would also be useful for those that are curious about what the script does, but don't want to read and fully understand the code.

**Comment by askolesov** [comment]
@WadeBarnes Great ideas

File: scripts/remove_ledger.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+from pathlib import Path
+import shutil
+from sys import argv
+from indy_common.config_util import getConfig
+from indy_common.config_helper import ConfigHelper
+
+
+def warn(ledger_name, directories_path):
+    print('The follow directories will be delete:')
```
Comment (WadeBarnes) [inline scripts/remove_ledger.py:11]:
```suggestion
    print('The follow directories will be deleted:')
```

File: scripts/remove_ledger.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    
+    for path in directories_path:
+        print(str(path))
+    
+    print('Delete ledger is irrevocable operation.\nProceed only if you know consequences.')
```
Comment (WadeBarnes) [inline scripts/remove_ledger.py:16]:
```suggestion
    print('Deleting a ledger is an irrevocable operation.\nProceed only if you know the consequences.')
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Looking good.  A few minor suggestions.

File: scripts/remove_ledger.py [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+from pathlib import Path
+import shutil
+from sys import argv
+from indy_common.config_util import getConfig
+from indy_common.config_helper import ConfigHelper
+
+
+def warn(ledger_name, directories_path):
+    print('The follow directories will be deleted:')
```
Comment (askolesov) [inline scripts/remove_ledger.py:11]:
```suggestion
    print('The following directories will be deleted:')
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
DCO sign-off is missing from your last commit.

**Comment by WadeBarnes** [comment]
(ic) test this please.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Code is failing static analysis; https://build.sovrin.org/job/indy-node/job/indy-node-verify-x86_64/2836/execution/node/18/log/

**Review by askolesov — APPROVED** [review]

**Comment by askolesov** [comment]
(ci) test this please

**Comment by askolesov** [comment]
(ci) ok to test

**Comment by askolesov** [comment]
@WadeBarnes It seems that all comments were processed. Could you review this again, please?

**Review by WadeBarnes — APPROVED** [review]

**Comment by WadeBarnes** [comment]
@Toktar, @skhoroshavin, We just need approval from a code owner.

**Review by Toktar — APPROVED** [review]

**Pull request merged**